server/container/guice/guice-common/src/main/java/org/apache/james/utils/ExtendedServerProbe.java

```diff
@@ -27,12 +27,13 @@
 import org.apache.james.cli.probe.ServerProbe;
 import org.apache.james.mailbox.exception.BadCredentialsException;
 import org.apache.james.mailbox.exception.MailboxException;
+import org.apache.james.mailbox.model.ComposedMessageId;
 import org.apache.james.mailbox.model.MailboxPath;
 import org.apache.james.mailbox.store.mail.model.Mailbox;
 
 public interface ExtendedServerProbe extends ServerProbe {
 
-    void appendMessage(String username, MailboxPath mailboxPath, InputStream message, Date internalDate, boolean isRecent, Flags flags) 
+    ComposedMessageId appendMessage(String username, MailboxPath mailboxPath, InputStream message, Date internalDate, boolean isRecent, Flags flags) 
```

Comment:
Aren't we supposed to remove this ComposedMessageId class when MessageId implementations are ok?